### DISCUSSION TITLE
How to mark if a time entry is already billed

### DISCUSSION THREAD

**Discussion by John-H-Smith** [solidtime-io/solidtime#247] [Q&A] [answered]
Often, within a large project, the customer will pay for reached milestones. How can I then mark a time entry as billed to not invoice the very same time entry multiple times?

**Comment by korridor** [comment, marked as answer]
Thanks for the question. There is currently not a specific feature for that use case, but you can use the tags to achieve that. You can create a tag "Billed" and assign the time entries to this tag once you billed them.
